### ISSUE TITLE
[Link and Image]Clean Up

### ISSUE THREAD

**Issue by evelyntorres1** [GSA/digitalgov.gov#1486]
## Issue

**Title** — **User Experience Impossible: The Line Between Accessibility and Usability**
**Demo URL** — https://demo.digital.gov//2014/11/17/user-experience-impossible-the-line-between-accessibility-and-usability/
**Live URL** — https://digital.gov//2014/11/17/user-experience-impossible-the-line-between-accessibility-and-usability/

---

**'assistive technology' link does not work.  goes to www.disability.gov but the page does not load.
Second image has 3 'expand in new window' icons surrounding the image. It looks strange and the images do not have captions throughout the article.
